apps/amazing/amazing.star

```diff
@@ -338,14 +340,16 @@ def next_start(visited):
                 if neighbour in visited:
                     return cell
 
+    return None
```

Comment:
:\

Reply:
This resolves a linter issue where there is an implicit return - this felt the safest way to resolve it?